tensorflow/stream_executor/stream.h

```diff
@@ -351,6 +355,32 @@ class Stream {
     return port::UnimplementedError("DNN library is not found.");
   }
 
+  template <typename InputType, typename OutputType>
+  port::Status ConvolveWithExecutionPlan(
+      const dnn::BatchDescriptor &input_descriptor,
+      const DeviceMemory<InputType> &input_data,
+      const dnn::FilterDescriptor &filter_descriptor,
+      const DeviceMemory<InputType> &filter_data,
+      const dnn::ConvolutionDescriptor &convolution_descriptor,
+      const dnn::BatchDescriptor &output_descriptor,
+      DeviceMemory<OutputType> *output, ScratchAllocator *scratch_allocator,
+      const dnn::AlgorithmConfig &plan_config,
+      dnn::ProfileResult *output_profile_result) {
+#if GOOGLE_CUDA
```

Comment:
Is this #if unncessary? Same for other similar ones.

Reply:
I think this one is still necessary. Since the implementation depends on `gpu::CudnnSupport` which is defined in cuda_dnn.h. For the cuda_dnn.h, it includes the `tensorflow/stream_executor/cuda/cuda_activation.h` which requires cuda config: e.g. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/stream_executor/cuda/BUILD#L389